controllers/util/tests/tests.go

```diff
@@ -0,0 +1,158 @@
+/**
+ * Copyright 2022 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package tests
+
+import (
+	"strings"
+
+	csiv1 "github.com/IBM/ibm-block-csi-operator/api/v1"
+	clustersyncer "github.com/IBM/ibm-block-csi-operator/controllers/syncer"
+	"github.com/IBM/ibm-block-csi-operator/pkg/config"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var (
+	nodeContainerName = clustersyncer.NodeContainerName
+	controllerContainerName = clustersyncer.ControllerContainerName
+	defaultControllerByName map[string]csiv1.IBMBlockCSIControllerSpec
+	defaultNodeByName map[string]csiv1.IBMBlockCSINodeSpec
+)
+var err = config.LoadDefaultsOfIBMBlockCSI()
```

Comment:
besides the existing name of `LoadDefaultsOfIBMBlockCSI`, I don't think "default" should be used (not in `defaultControllerByName`, not in `setDefaultNodeImageByName`, etc...), because in the envtest context, the cr yaml is not used to determine "default" values (as the operator uses it), but the cr yaml is used just to apply some ("test input+expected") values

Reply:
done